package.json

```diff
@@ -17,6 +17,7 @@
     "url": "git://github.com/angular/material.git"
   },
   "private": true,
+  "main": "./dist/angular-material.js",
```

Comment:
Why should we have a main file here? ngMaterial doesn't run inside of NodeJS.

Also a "main" with an associated index file, is already provided in `bower-material`.
> https://github.com/angular/bower-material/blob/master/package.json